inch.go

```diff
@@ -313,21 +321,31 @@ func (s *Simulator) generateBatches() <-chan []byte {
 			fields = append(fields, []byte(fmt.Sprintf("v%d=1%s", i, delim))...)
 		}
 
+		// Size internal buffer to consider mx+tags+ +fields.
+		buf := bytes.NewBuffer(make([]byte, 0, 2+len(tags)+1+len(fields)))
+
+		// Write points.
+		var lastMN int
+		lastM := []byte("m0")
 		for i := 0; i < s.PointN(); i++ {
-			// Write point.
-			buf.Write([]byte(fmt.Sprintf("m%d", i%s.Measurements)))
+			lastMN = i % s.Measurements
+			lastM = append(lastM[:1], []byte(strconv.Itoa(lastMN))...)
+			buf.Write(lastM) // Write measurement
+
 			for j, value := range values {
-				fmt.Fprintf(&buf, ",tag%d=value%d", j, value)
+				tags = append(tags, fmt.Sprintf(",tag%d=value%d", j, value)...)
```

Comment:
Might be fewer allocations to use a couple calls to strconv.AppendInt here too. May not be worth the time to benchmark.